dat/missions/zalek/neburesearch/neburesearch_00.lua

```diff
@@ -0,0 +1,221 @@
+--[[
+   
+   Mission: Novice Nebula Research
+   
+   Description: A Za'lek student asks the player to help him with his reasearch. The player has to visit Doeston and Arandon.
+   Some minor complications on the way have to be expected.
+   
+   Difficulty: Easy
+
+]]--
+
+include "dat/scripts/numstring.lua"
+
+
+t_sys = {}
+t_sys[1] = "Doeston"
+t_sys[2] = "Arandon"
+
+-- localization stuff, translators would work here
+bar_desc = _("You see a young scientist talking with some pilots, apparently without success.")
+mtitle = {}
+mtitle[1] = _("Novice Nebula Research")
+misn_reward = _("%s credits and the gratitude of a student")
+mdesc = {}
+mdesc[1] = _("Go to the %s system.")
+title = {}
+title[1] = _("Bar")
+title[2] = _("Please...")
+title[3] = _("What a mess...")
+title[4] = _("Ready")
+title[5] = _("Technical problems")
+title[6] = _("Shadows in the darkness")
+title[7] = _("Mission Success")
```

Comment:
This is minor and you don't need to change this since you already did it this way, but I just want to recommend against this old style that used to be encouraged of throwing everything into ordered tables. It makes the names non-descriptive and confusing. Like I said, no need to change this here, but I just wanted to strongly recommend avoiding this style for future missions.

🕵️